geth/jail/jail.go

```diff
@@ -46,17 +46,39 @@ func (jail *Jail) BaseJS(js string) {
 }
 
 // NewJailCell initializes and returns jail cell
-func (jail *Jail) NewJailCell(id string) common.JailCell {
+func (jail *Jail) NewJailCell(id string) (common.JailCell, error) {
```

Comment:
If we do so, we will not be able to meet the `JailManager` interface.  See https://github.com/status-im/status-go/blob/77d01c424e5dab818ce07cff38bdd9779ecdc2c8/geth/common/types.go#L194